src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: $card-width;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: $border-radius;
+  background-color: #fff;
+  transition: all 0.3s;
```

Comment:
Avoid using `all` in the `transition` property. It is not performant as it will trigger transitions for all properties, including those that do not need to change. Specify which properties you want to transition, such as `transform` for the hover effect.